src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
+@import '//fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    margin-inline: 3px;
+    background-image: url(../images/imac.jpeg);
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  &__text {
```

Comment:
this element doesn't make a lot of sense since it doesn't have its own styles
you can create 2 separate elements (instead of `head `and `gray`) modificators